src/common/mailer/mailer.controller.ts

```diff
@@ -0,0 +1,41 @@
+import { Controller, OnModuleInit, Param, Post, Res } from "@nestjs/common";
```

Comment:
Instead of putting it inside the Mailer manager, move the Substrate code here. Make them private, so it cannot be accessed externally.

Reply:
Revision: Remove the `mailer.controller.ts` and create a new folder named `Email` to house the new API endpoints.

Reply:
Don't make a `controller` inside the `common` folder.